target_chains/ethereum/contracts/forge-test/Pyth.WormholeMerkleAccumulator.t.sol

```diff
@@ -205,6 +271,14 @@ contract PythWormholeMerkleAccumulatorTest is
         pyth.updatePriceFeeds{value: updateFee}(updateData);
 
         assertPriceFeedMessageStored(priceFeedMessages1[0]);
+
+        bytes32[] memory priceIds2 = new bytes32[](1);
+        priceIds2[0] = priceFeedMessages1[0].priceId;
+
+        PythStructs.PriceFeed[] memory priceFeeds2 = pyth.parsePriceFeedUpdates{
+            value: updateFee
+        }(updateData, priceIds2, 0, MAX_UINT64);
+        assertParsedPriceFeedStored(priceFeeds2[0]);
```

Comment:
doesn't this need to test that priceFeeds2 has length 1 also?

also if you swap the order of the updates in updateData, the code should return the other update? I think it's worth testing that behavior too.